src/rgw/rgw_op.h

```diff
@@ -1065,7 +1065,7 @@ class RGWGetACLs : public RGWOp {
 
 class RGWPutACLs : public RGWOp {
 protected:
-  size_t len;
+  int len;
```

Comment:
It seems strange to change size_t to int for an I/O operation. What's the reasoning? 
Also, I'm not at this point sure where all of this goes, but consider default initializing it, ie.:
int len = 0; 
(Same with the data pointer, etc.-- again, I don't know where this is going, or how it's used yet, so if that's not appropriate, please disregard.)